client/tests/unit/serializers/package-test.js

```diff
@@ -0,0 +1,23 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Serializer | package', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    const store = this.owner.lookup('service:store');
+    const serializer = store.serializerFor('package');
+
+    assert.ok(serializer);
+  });
+
+  test('it serializes records', function(assert) {
+    const store = this.owner.lookup('service:store');
+    const record = store.createRecord('package', {});
+
+    const serializedRecord = record.serialize();
+
+    assert.ok(serializedRecord);
```

Comment:
What does this assertion test?  Just that there's no errors when `record.serialize()` is invoked?

Reply:
Yep! Not a very good test.

Reply:
I am removing these serializers in a later PR. They don't need to be there.

Reply:
Cool - was just checking as I'm trying to understand/learn test writing.